tensorflow/lite/tools/cmake/modules/eigen.cmake

```diff
@@ -19,6 +19,10 @@ endif()
 
 include(OverridableFetchContent)
 
+if(QNX)
+  set(PATCH_CMD git reset --hard && git apply "${QNX_PATCH_DIR}/eigen.patch")
```

Comment:
Is this our standard way of doings so? I don't know our standardized way.... Can we change the repo link to our repo instead?

Reply:
It's already pointing at our repo (OverridableFetchContents lets you override git url and tag but not the patch command) from inside of Build Files. I'm rewriting this to pass the patch command through Build Files too.

Reply:
I refactored out the patches in source. Let me know if you can think of a better approach.